bin/dummy_mentor

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+questionnaire=$1
+default_page=index.html
+last_page=final.html
+questionnaire_page=questionnaire.html
+
+cd public/dummy
```

Comment:
eerst mkdir -p (oid public/dummy) ? Want straks staat die directory niet meer in de repo toch? Dus dan gaat dit commando fout bij een freshe checkout.

Reply:
done